tests/test_public_cli.py

```diff
@@ -1400,3 +1400,68 @@ def test_plot_bad_nulls(
         "WARNING: Cannot plot hadamard as matrix contains 2 nulls (out of 2²=4 guessing comparisons)\n"
         in stderr
     ), stderr
+
+
+def test_classify_failures(tmp_path: str) -> None:
+    """Check classify failures."""
+    tmp_dir = Path(tmp_path)
+
+    with pytest.raises(
+        SystemExit, match="ERROR: Database /does/not/exist does not exist"
+    ):
+        public_cli.classify(database=Path("/does/not/exist"), outdir=tmp_dir)
+
+    with pytest.raises(
+        SystemExit, match="ERROR: Output directory /does/not/exist does not exist"
+    ):
+        public_cli.classify(database=":memory:", outdir=Path("/does/not/exist/"))
+
+
+def test_classify(
+    capsys: pytest.CaptureFixture[str], tmp_path: str, input_genomes_tiny: Path
+) -> None:
+    """Check working example of classify."""
+    tmp_dir = Path(tmp_path)
+    tmp_db = tmp_dir / "classify_complete.sqlite"
+    session = db_orm.connect_to_db(tmp_db)
+    config = db_orm.db_configuration(
+        session, "fastANI", "fastani", "1.2.3", create=True
+    )
+
+    fasta_to_hash = {
+        filename: file_md5sum(filename)
+        for filename in sorted(input_genomes_tiny.glob("*.f*"))
+    }
+    for filename, md5 in fasta_to_hash.items():
+        db_orm.db_genome(session, filename, md5, create=True)
+
+    # Record all of the possible comparisons, leaving coverage null
+    genomes = list(fasta_to_hash.values())
+    for query_hash in genomes:
+        for subject_hash in genomes:
+            db_orm.db_comparison(
+                session,
+                config.configuration_id,
+                query_hash,
+                subject_hash,
+                1.0 if query_hash is subject_hash else 0.99,
+                12345,
+                cov_query=1.0 if query_hash is subject_hash else 0.88,
+            )
+
+    db_orm.add_run(
+        session,
+        config,
+        cmdline="pyani guessing ...",
+        fasta_directory=input_genomes_tiny,
+        status="Complete",
+        name="Test classify when all data present",
+        fasta_to_hash=fasta_to_hash,
+    )
+
+    public_cli.classify(database=tmp_db, run_id=1, outdir=tmp_dir)
```

Comment:
If you leave out the run ID this will test the code picking the latest run (something noted on the test coverage as missing).